services/paymaster/README.md

```diff
@@ -50,13 +50,13 @@ docker run --name jaeger \
 The crate also exposes a cli tool to validate arbitrary transactions against a specified config. You can run this via the following command:
 
 ```
-cargo run --bin paymaster-tx-validator validate -c <CONFIG_PATH> --transaction-hash <ONCHAIN_TRANSACTION_HASH>
+cargo run --bin paymaster-tx-validator validate -c <CONFIG_PATH> --rpc-http-url <RPC_HTTP_URL> --rpc-ws-url <RPC_WS_URL> --transaction-hash <ONCHAIN_TRANSACTION_HASH>
 ```
 
 Alternatively, you can provide a serialized transaction as a base64 string via the `--transaction` argument in place of the hash. Additionally, you could ask the tool to validate a specified number of the most recent transactions that interacted with this domain's sponsor pubkey via:
 
 ```
-cargo run --bin paymaster-tx-validator validate -c <CONFIG_PATH> --domain <DOMAIN> --recent-sponsor-txs <NUMBER_OF_RECENT_TXS>
+cargo run --bin paymaster-tx-validator validate -c <CONFIG_PATH> --rpc-http-url <RPC_HTTP_URL> --rpc-ws-url <RPC_WS_URL> --domain <DOMAIN> --recent-sponsor-txs <NUMBER_OF_RECENT_TXS>
```

Comment:
Since `pasymaster-tx-validator` doesn't use websocket connection afaik, it seems more user friendly to encourage using it without providing the `--rpc-ws-url` argument